scripts/automator/packages/manager-apt

```diff
@@ -1,2 +1,2 @@
 # Package repo: https://packages.ubuntu.com/
-packages+=(ccache libtool build-essential autotools-dev automake libsdl2-dev libsdl2-net-dev libncurses-dev libopusfile-dev)
+packages+=(ccache libtool build-essential autotools-dev automake libsdl2-dev libsdl2-net-dev libncurses-dev libopusfile-dev libfluidsynth-dev)
```

Comment:
Found fluidsynth 2.0 header + libs provided for our other package manager files too:

**Haiku**
- package name: "fluidsynth2_devel"
- link: https://depot.haiku-os.org/#!/pkg/fluidsynth2_devel/haikuports/2/1/1/-/2/x86_64?bcguid=bc221-HURS

**MSYS2:**
  - "mingw-w64-i686-fluidsynth" https://packages.msys2.org/package/mingw-w64-i686-fluidsynth?repo=mingw32
  - "mingw-w64-x86_64-fluidsynth" https://packages.msys2.org/package/mingw-w64-x86_64-fluidsynth?repo=mingw64

**Pacman**
- package name: "fluidsynth"
- link: https://www.archlinux.org/packages/extra/x86_64/fluidsynth/

**Zypper**
- package name: "fluidsynth-devel"
- link: https://opensuse.pkgs.org/15.2/multimedia-libs-x86_64/fluidsynth-devel-2.1.4-lp152.86.1.x86_64.rpm.html